docs/framework/20categories/20architecture/token-bridges.md

```diff
@@ -1,22 +1,38 @@
-at a later time. The invariant such protocols must preserve is that the value 
-of an asset only resides in one place at a given time and that the total 
-supply of synthetic assets matches the total underlying. Scenarios in 
-which either locked assets on a source network are stolen or synthetic 
-assets are arbitrarily minted on a destination network, break these 
-invariants. This creates a perpetual risk for token holders of 
-synthetic assets, in which these assets could become worthless 
-in the event of such failures.
-
-Moreover, lock-and-mint token bridges in particular create 
-honey pots that make them an ideal target for malicious 
-actors. A significant portion of bridge hacks over the 
-last couple of years has involved this source of risk.
+One of the most common cross-chain use cases is the [transfer of assets](../../01intro/introduction.md#types-of-crosschain-interaction) from one network to another. This enables an asset in one network to be used inside applications in another. For instance, a user might want to transfer her ETH from Ethereum to Avalanche so she can use it as collateral in a lending protocol on Avalanche. A token bridge is a protocol that enables this requirement. It often relies on an underlying messaging protocol for cross-chain message communication. 
+
+A core security requirement of such a protocol is preventing cross-chain double-spends by ensuring that the original asset's value can only be realized in one network. There are two models for enabling this requirement, depending on the characteristics of the underlying asset.
+
+#### Lock and Mint
+In this model, token bridges mimic the transfer of an asset by locking the original asset in the source network and issuing synthetic representations on the destination. The original asset remains encumbered for as long as a synthetic token exists in the destination network. A user can redeem the original asset by burning the corresponding synthetic token in the destination network. 
+
+This approach is commonly employed today because it is the only viable model for transferring assets to a network they are not natively issued in. For instance, moving the native assets of layer-one protocols (e.g., ETH, SOL, AVAX) to different chains necessitates this model. 
+
+Such protocols must preserve the following core security invariants at all times:
+
+1. Every synthetic token has a corresponding locked asset in the source network.
+1. There can only be a single outstanding claim (i.e., synthetic token) against a locked asset.  
+
+Thus the total supply of synthetic tokens across networks must match the number of assets locked in a source network. If these invariants fail, synthetic tokens can become worthless. Because the synthetic token is effectively an IOU issued by the token bridge to the [token holder](../../10stakeholders/stakeholders.md#stakeholders), the holder carries perpetual exposure to such risks. 
```

Comment:
I think it would be good to introduce the term "unbacked" asset. How about:

If these invariants fail, the synthetic tokens become _partially unbacked_, meaning that each asset on the destination network is not backed by an asset on the source network. In extreme cases, where no assets on the source network are backing the assets on the destination network, the synthetic tokens are worthless.

Reply:
good suggestion. added.